docs/CONTRIBUTING.md

```diff
@@ -59,91 +59,9 @@ please create an issue on the GitHub repository:
   include that information in your description of the issue
   or in a comment to the issue.
 
-## Building the documentation locally
+## [Building the documentation locally](https://github.com/keptn/lifecycle-toolkit/blob/main/docs/builddocslocally.md)
```

Comment:
We should not hide links in Headings - if we want to direct people to the new document, we should do this in a proper format. eg. 

```markdown
Further information:
- building the documentation locally
- Building markdown files without Hugo

```

and we should link to the appropriate section. Not just the page. Additionally, those pages will get rendered with Hugo - so absolute links will be wrong. Hugo means that they're rendered and will be displayed on https://lifecycle.keptn.sh, and we do not want to direct people away from this page to GitHub. so please use relative links.

TODOS:
- [ ] use relative links
- [ ] do not use headings for lists

Reply:
hey @thschue, @aepfli thankyou reviewing and as per your notes, I've changed the files. Hope that resolves everything..